### PR TITLE
cannot retrieve binance tickers due to missing timestamp

### PR DESCRIPTION
**Pull request by ed50245**

### CODE REVIEW

File: spring-boot-starter/autoconfigure/src/main/java/tech/cassandre/trading/bot/dto/market/TickerDTO.java [hunk 1]
```diff
@@ -87,28 +87,4 @@ public CurrencyDTO getQuoteCurrency() {
             return null;
         }
     }
-
```
Comment (straumat) [inline spring-boot-starter/autoconfigure/src/main/java/tech/cassandre/trading/bot/dto/market/TickerDTO.java:90]: I cannot use lomboz equals as I have my own equalsbuilder. Lomboz  makes a .equals() but on zonedDateTime the same moment is not equal if timzeone are different

**Review by straumat — CHANGES_REQUESTED** [review]

**Review by straumat — CHANGES_REQUESTED** [review]
Sorry i can't accept that, as the equals method doesn't work well with ZonedDateTime. It doesn't compare moment but also timzeone. The same moment is not equals if you don't have the same timeZone.
Like for bigdecimal, it also compares format

File: spring-boot-starter/autoconfigure/src/main/java/tech/cassandre/trading/bot/dto/market/TickerDTO.java (hunk 1 above)
Comment (ed50245) [reply spring-boot-starter/autoconfigure/src/main/java/tech/cassandre/trading/bot/dto/market/TickerDTO.java:90]: would you have different timezones on different tickers?
From what I see the timestamp is converted to system timezone for every ticker

Comment (straumat) [reply spring-boot-starter/autoconfigure/src/main/java/tech/cassandre/trading/bot/dto/market/TickerDTO.java:90]: Some peoples are asking me to store tickers and, like for bigdecimal, the value stored in database is not necessarily the same that the value saved in Java. So the value retrieved would be different. I think it's better to have a solution in mapstruct. Do you want me to do it ?

Comment (ed50245) [reply spring-boot-starter/autoconfigure/src/main/java/tech/cassandre/trading/bot/dto/market/TickerDTO.java:90]: I don't have time in the following days, so if you can, please.
Anyway, did not knew about ZonedDateTime and just took a look:

`  @Override
    public boolean equals(Object obj) {
        if (this == obj) {
            return true;
        }
        if (obj instanceof ZonedDateTime) {
            ZonedDateTime other = (ZonedDateTime) obj;
            return dateTime.equals(other.dateTime) &&
                offset.equals(other.offset) &&
                zone.equals(other.zone);
        }
        return false;
    }`

Seems it does check equality of the offset and zone.

**Pull request closed without merging**